### PR TITLE
Add Bower to README install instructions

### PR DESCRIPTION
**Pull request by feimosi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Bulma is in early but active development! Try it out now:
 npm install bulma
 ```
 
+### Bower
+
+```sh
+bower install bulma
```
Comment (jbrooksuk) [inline README.md]: Maybe `bower install --save bulma`

Comment (feimosi) [inline README.md]: I know, but I wanted to stay consistent with the `npm` part

Comment (jbrooksuk) [inline README.md]: Which should be updated too IMHO.

Comment (feimosi) [inline README.md]: That stays up to @jgthms. I can do that on a request.

**Comment by jgthms** [comment]
At first, Bulma only used bower. But people suggested that npm was the way to go now.

I left the bower support as legacy but intend to remove it completely very soon, as even the creator of bower thought npm was a better option.

**Comment by feimosi** [comment]
@jgthms I can understand your point, but I really wouldn't remove it. There're plenty of projects that are forced to use bower (I know a few) and it doesn't cost you any effort to keep it. 
With bower, you have that convenience, it automatically updates the package based on Git tags.

**Comment by jbrooksuk** [comment]
I actively use Bower :)

**Pull request merged**